Source/JavaScriptCore/dfg/DFGByteCodeParser.cpp

```diff
@@ -4363,6 +4363,12 @@ bool ByteCodeParser::handleIntrinsicGetter(Operand result, SpeculatedType predic
         return true;
     }
 
+    case SpeciesGetterIntrinsic: {
+        insertChecks();
+        set(result, thisNode);
```

Comment:
I think this needs `ToThis`.

Reply:
Correction: since `thisNode` is coming from `handleGetById`, it's guaranteed to be "sane" (not JSScope / JSGlobalObject), so this is correct!